docs/proposal-component-names.md

```diff
@@ -0,0 +1,60 @@
+# Proposal (Feb 2020)
+
+## Cylc 8 System Component Naming
+
+Cylc is now a complex multi-component system. It is important that we settle on
+consistent, unambiguous, and where possible standard, component names so that
+users and developers can communicate without confusion. (This was not easy even
+before Cylc 8!).
+
+## Proposed Final Names
+
+Note we have already agreed to drop **suite** for **workflow** ("suite" is
+non-standard outside of the NWP community).
+
+| Component Name               | Code Repository   | Config File                                          | Description |
+| --                           |----               | ---                                                  |---          |
+|:boom: **Cylc Scheduler**     | cylc-flow         | :boom: `cylc-flow.conf` and `cylc-flow-global.conf`  | an instance of the cylc-flow scheduler program, to manage a single workflow | 
+|**Cylc Hub**                  | (Jupyter Hub)     | :boom: `<cylc-location>/jupyterhub_config.py`               | the single point of access that users log in to |
```

Comment:
```diff
- `<cylc-location>/jupyterhub_config.py`
+ hub.py
```

Reply:
I'll note this in the proposal as an option (I did originally suggest `cylc-hub.py`) but I think leaving it as  `jupyterhub_config.py` in the cylc-specific location would be fine - no reason to hide the fact that "Cylc Hub" is in fact just JupyterHub, and users won't normally have to deal directly with hub config.

Reply:
We have control of this file / search path so we can call it what we want but fair point.